app/src/main/java/io/appium/settings/ForegroundService.java

```diff
@@ -72,19 +50,7 @@ public int onStartCommand(Intent intent, int flags, int startId) {
     }
 
     private void startForegroundService() {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-            createChannel();
-        }
-        NotificationCompat.BigTextStyle bigTextStyle = new NotificationCompat.BigTextStyle();
-        bigTextStyle.setBigContentTitle(CHANNEL_NAME);
-        bigTextStyle.bigText(CHANNEL_DESCRIPTION);
-        Notification notification = new NotificationCompat.Builder(this, CHANNEL_ID)
-                .setStyle(bigTextStyle)
-                .setWhen(System.currentTimeMillis())
-                .setSmallIcon(R.drawable.ic_launcher)
-                .setLargeIcon(BitmapFactory.decodeResource(getResources(), R.drawable.ic_launcher))
-                .build();
-        startForeground(1, notification);
+        startForeground(1, NotificationHelpers.getNotification(this));
```

Comment:
I would say it makes sense to put the notification id into a constant

Reply:
ok, I'll add constant in NotificationHelper